docker/arm-server/Dockerfile

```diff
@@ -1,30 +1,53 @@
-FROM ubuntu:22.04
-
+# Docker image for running CF instances in ARM64 server.
+# Docker image includes HO(Host Orchestrator) inside,
+# so it could execute CF instance with API in HO.
+FROM ubuntu:22.04 AS cuttlefish-arm64
+
+# Expose Operator Port (HTTP:1080, HTTPS:1443)
+EXPOSE 1080 1443
+# Expose HO(Host Orchestrator) Port (HTTP:2080, HTTPS:2443)
+EXPOSE 2080 2443
```

Comment:
I think there are conflicts with https://cs.android.com/android/platform/superproject/main/+/main:device/google/cuttlefish/tools/launch_cvd_arm64_server_docker.sh;l=83?q=launch_cvd_arm
 because I uses 2443 for remote web ui, please modify the script as well.

Reply:
I think we don't need to change. Regardless it's operator or HO, both supports WebUI with operator's API. Was ther e any error when using WebUI of HO when running the script?

Reply:
i mean current logic set up port forward rule like 2443(remote) -> xxxxx(randomly assigned in host) -> 1443(docker), but if we expose 2443 as well, there will be two rule for 2443. so we need to modify 2443 in the script to the other port

Reply:
AFAIU, `EXPOSE` in Dockerfile doesn't expose port via `-p` option, and it's exposed with randomly assigned port for `-P` option.

Just for my understanding on `tools/launch_cvd_arm64_server_docker.sh`, if we expose port 2443, does it try to make port forwarding via ssh with same port like below? Unless, I don't think we need to modify the script.
- 2443(remote) -> xxxxx(randomly assigned in host) -> 2443(docker)
- 2443(remote) -> xxxxx(randomly assigned in host) -> 1443(docker)

Reply:
Thought again, I changed my mind. Regardless exposing 2443 or not, we should edit the script since we should consider machine using the script already have HO inside.

Reply:
Done, and submitted into AOSP. Let this be resolved.